src/log.c

```diff
@@ -54,7 +54,7 @@ void init_logging(void)
 	is_a_tty = isatty(STDOUT_FILENO);
 
 	pthread_mutexattr_init(&mutexattr);
-#ifndef __APPLE__
+#ifdef HAVE_PTHREAD_MUTEXATTR_SETROBUST
```

Comment:
Excellent! It would be great if all \_\_APPLE\_\_ ifdef's could eventually be changed the same way.